bookserver/models.py

```diff
@@ -348,7 +351,23 @@ class AuthUser(Base, IdMixin):
     accept_tcp = Column(Web2PyBoolean)
 
 
-AuthUserValidator = sqlalchemy_to_pydantic(AuthUser)
+BaseAuthUserValidator = sqlalchemy_to_pydantic(AuthUser)
+
+
+class AuthUserValidator(BaseAuthUserValidator):
```

Comment:
Here's an example of a mypy that I have no idea what to do with.

```
bookserver/models.py:357: error: Variable "bookserver.models.BaseAuthUserValidator" is not valid as a type
bookserver/models.py:357: note: See https://mypy.readthedocs.io/en/latest/common_issues.html#variables-vs-type-aliases
bookserver/models.py:357: error: Invalid base class "BaseAuthUserValidator"
```
The validator runs and works fine.

Reply:
@bjones1 any ideas on how to make these mypy errors go away?

Reply:
I made them go away with the ignore.

Reply:
No idea on this one -- I played a bit, but ignoring is the best alternative I see for now.